### ISSUE TITLE
Feature Request: Detect new event when added to Google Calendar

### ISSUE THREAD

**Issue by clandestine-avocado** [ljmerza/calendar-card#78]
Title says it all - would love this as a feature - which would enable notifications containing the event name, start and finish time/date etc.

**Comment by ljmerza** [comment]
added in `v3.107.0`. Select a notify entity via `notifyEntity` and a datetime format if desired via `notifyDateTimeFormat`

**Comment by clandestine-avocado** [comment]
Sorry - I'm not following. Where would I make these selections? I don't see anything in the card UI, or as a service in HA. Apologies if this is obvious!

**Comment by ljmerza** [comment]
in the card editor there is a dropdown to select what notify entity you want to use or you can set it manually with yaml

**Comment by clandestine-avocado** [comment]
I see it now in the UI...I had failed to upgrade, my bad. Is there any way to customize the message if I did it in YAML? 

Background info on "why", if you're interested:

I'm trying to replace a more complicated (an error prone) string of events that gets new Google cal events into my Outlook cal, which I cannot connect to directly, for reasons that don't matter here.
I want to replace this with something from HA. 

Right now it goes like this:

- [ ] IFTTT listens for a new Google calendar event. 
- [ ] When discovered, it sends me a notification (bc IFTTT Gmail integration is broken)
- [ ] The body of the notification is formatted like this, with delimiters
- [ ] _#newevent | Test event subject | location | January 27, 2020 at 09:00PM | January 27, 2020 at 10:00PM_
- [ ] Tasker on my phone listens for the notifications with the hashtag #newevent
- [ ] Tasker then forwards it, all in the subject line of an email, to my work email
- [ ] I then process it into my work calendar with VBA

So, if I could get HA to customize the notify message, to allow for prefixes and delimiters, and get it all in the subject line, I could take a 7 step process down to 2 steps. Its actually worse than this because I have to set it up for each calendar! Anyway, thanks for reading, hope this is possible on another version!
